### ISSUE TITLE
Naming convention

### ISSUE THREAD

**Issue by pnutus** [Liffon/otp#3]
Are we sticking with camelCase for variables? Right now it's not_reallyConsistent.

(Maybe vim has a fancy command for converting to camel case?)

**Comment by Liffon** [comment]
Yes, I think camelCase is what I originally intended. It shall be fixed!

tors 21 maj 2015 kl 23:49 skrev Pontus Granström notifications@github.com:
